deploy-eks/fb-editor-chart/templates/deployment.yaml

```diff
@@ -133,6 +133,19 @@ spec:
               secretKeyRef:
                 name: fb-editor-secrets-{{ .Values.environmentName }}
                 key: pingdom_alert_integration_id
+{{- if eq .Values.from_address "enabled" }}
+          - name: AWS_SES_SECRET_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                name: fb-editor-secrets-{{ .Values.environmentName }}
+                key: aws_ses_secret_access_key
+          - name: AWS_SES_ACCESS_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                name: fb-editor-secrets-{{ .Values.environmentName }}
```

Comment:
yeah this might fail when going to Live as there are no creds for Live in fb-editor-deploy